README.md

```diff
@@ -197,94 +182,137 @@ coastal area.
+
+**Inductiva API** provides a simple way to manage the machines used to run the simulations. Users can launch machines, list the available machines and terminate them.
+In this way, users do not need to wait for their simulations in a queue and can have full control of the hardware used.
 
-The user must provide the path for the PDB file (pdb_file) corresponding to the protein to be simulated. Aditionally, one can specify the temperature (in Kelvin), which defaults to 300 K. 
+**Inductiva API** simplifies simulation at scale.
 
-After the initialization, we are ready to run the simulation:
+Start your machines and run your simulations:
 
 ```python
-output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
-```
+
+import inductiva
+
+machine = inductiva.resources.launch_machines(name="test_machine",
+                                              machine_type="c2-standard-16")
+
+# Example with ProteinSolvation scenario
+scenario = molecules.scenarios.ProteinSolvation(pdb_file, temperature = 300)
+
 output = scenario.simulate(simulation_time = 10,
-                           nsteps_minim = 5000)
-The simulate method initializes the simulation. In this call, we can set the simulation duration (in ns) and the number of steps for the energy minimization. 
+                           nsteps_minim = 5000,
+                           resources=machine)
+```
 
-Visualize the results: 
+But do not forget to kill your machines:
 ```python
-view = ouptut.render_interctive(representation= "ball+stick", add_backbone=True)
+
+machine.kill()
 ```
 
-This yields an interactive visualization of the protein's trajectory that can be visualized and manipulated in a standard jupyter notebook. The user can specify the representation used for the protein and choose to add the backbone to the visualization.  
-<p align="center">
-  <img width="900" height="300" src="https://github.com/inductiva/inductiva/assets/114397668/d8eb9c0b-8809-4da6-97a1-3448f2809af3">
-</p>
+## Async API
+
+Up until now, all examples have run synchronously, which allows users to get feedback about the simulation while it is running. However, this is not always the best option. For example, if the user wants to run a large number of simulations, it is better to run them asynchronously. This way, the user can launch all the simulations and then check the results when they are ready.
+
+Let's look at an example with the wind tunnel:
+
+```python
+from inductiva import fluids
 
+scenario = fluids.scenarios.WindTunnel(
+    flow_velocity=[30, 0, 0],
+    domain_geometry={"x": [-6, 12], "y": [-5, 5], "z": [0, 10]})
+
+# Path to a folder containing a set of objects
+objects_path = ["object_1.obj", "object_2.obj", ..., "object_1000.obj"]
+
+for object in objects_path:
+    task = scenario.simulate(object_path=object,
+                      simulation_time = 100,
+                      output_time_step = 50,
+                      resolution = "medium",
+                      async=True)
```

Comment:
This parameter is called `run_async` as per https://github.com/inductiva/inductiva/pull/404.